src/fsharp/PostInferenceChecks.fs

```diff
@@ -1639,7 +1639,12 @@ and CheckDecisionTreeTest cenv env m discrim =
     | DecisionTreeTest.ActivePatternCase (exp, _, _, _, _, _) -> CheckExprNoByrefs cenv env exp
     | DecisionTreeTest.Error _ -> ()
 
-and CheckAttrib cenv env (Attrib(_, _, args, props, _, _, _)) = 
+and CheckAttrib cenv env (Attrib(tcref, _, args, props, _, _, m)) =
+    if [
```

Comment:
This creates a list each time the function is called. It's no big deal for a list of length 2, but you can pre-create the list in TcGlobals if you like.